### PR TITLE
test: add tests to `blas/ext/base/dnannsumors`

### LINKED ISSUE #3253: [RFC]: Incomplete code coverage for `blas/ext/base/dnannsumors`
### Description

This RFC proposes to add a test case in each of the relevant test files. The test case should handle the situation where the stride is 0 and the first element is NaN.

Coverage report: https://coverage.stdlib.io/blas/ext/base/dnannsumors/index.html

For reference see: #3051 

### Related Issues

Related issues # , # , and # .

### Questions

No.

### Other

No.

### Checklist

- [x] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [x] Searched for existing issues and pull requests.
- [x] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by saurabhraghuvanshii**
Resolves #3253 .

## Description

> What is the purpose of this pull request?

This pull request:

-   Add tests to blas/ext/base/dnannsumors

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #3253 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.dnannsumors.js [hunk 1]
```diff
@@ -232,6 +232,23 @@ tape( 'if provided a `stride` parameter equal to `0`, the function returns a sum
 	t.end();
 });
 
+tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.dnannsumors.js]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is `NaN`, the function returns `0.0`', function test( t ) {
```

Updating to be consistent with L122.

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.dnannsumors.native.js [hunk 2]
```diff
@@ -241,6 +241,23 @@ tape( 'if provided a `stride` parameter equal to `0`, the function returns a sum
 	t.end();
 });
 
+tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', opts, function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.dnannsumors.native.js]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is `NaN`, the function returns `0.0`', opts, function test( t ) {
```

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.ndarray.js [hunk 3]
```diff
@@ -232,6 +232,23 @@ tape( 'if provided a `stride` parameter equal to `0`, the function returns a sum
 	t.end();
 });
 
+tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.ndarray.js]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is `NaN`, the function returns `0.0`', function test( t ) {
```

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.ndarray.native.js [hunk 4]
```diff
@@ -241,6 +241,23 @@ tape( 'if provided a `stride` parameter equal to `0`, the function returns a sum
 	t.end();
 });
 
+tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', opts, function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.ndarray.native.js]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is `NaN`, the function returns `0.0`', opts, function test( t ) {
```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @saurabhraghuvanshii!

**Comment by saurabhraghuvanshii** [comment]
@kgryte welcome sir

**Pull request merged**